### PR TITLE
Sunshine Spam is visible

### PR DESCRIPTION
**Pull request by Raemon**
## Description of the PR

[Description here]

## Checklist for reviewing Pull Requests:

[ ] Reviewer has pulled branch and tried out functionality
[ ] Reviewer has mentally stepped through the code

*Checks for common errors:*

[ ] All variables are accessed in a null-safe way, or marked as required in the comments or prop-types
[ ] Loading states and exit conditions in methods and components are checked as early as possible and cause an early return
[ ] All calls to `collection.findOne` and `collection.find` or similar calls are prefixed by `await`

### CODE REVIEW

File: packages/lesswrong/lib/collections/comments/views.js [hunk 1]
```diff
@@ -222,6 +222,17 @@ ensureIndex(Comments, {legacyId: "hashed"});
 // Used in scoring cron job
 ensureIndex(Comments, {inactive:1,postedAt:1});
 
+Comments.addView("sunshineNewUsersComments", function (terms) {
+  return {
+    selector: {
+      userId: terms.userId,
+      $or: null,
```
Comment (jimrandomh) [inline packages/lesswrong/lib/collections/comments/views.js]:
```suggestion
      // Don't hide deleted
      $or: null,
```

File: packages/lesswrong/lib/collections/comments/views.js [hunk 2]
```diff
@@ -222,6 +222,17 @@ ensureIndex(Comments, {legacyId: "hashed"});
...
+    },
+    options: {sort: {postedAt: -1}, limit: terms.limit || 5},
+  };
+});
+// TODO – ask Jim about indices
```
Comment (jimrandomh) [inline packages/lesswrong/lib/collections/comments/views.js]: Yep, this needs its own index.

File: packages/lesswrong/lib/collections/posts/views.js [hunk 3]
```diff
@@ -549,6 +549,7 @@ Posts.addView("communityResourcePosts", function () {
 Posts.addView("sunshineNewPosts", function () {
   return {
     selector: {
+      status: null,
```
Comment (jimrandomh) [inline packages/lesswrong/lib/collections/posts/views.js]: What does `status: null` do?

File: packages/lesswrong/lib/collections/posts/views.js [hunk 4]
```diff
@@ -560,7 +561,31 @@ Posts.addView("sunshineNewPosts", function () {
...
+    },
+    options: {
+      sort: {
+        createdAt: -1,
+      }
+    }
+  }
+})
+ensureIndex(Posts,
+  augmentForDefaultView({ status:1, userId:1, reviewedByUserId:1, frontpageDate: 1, authorIsUnreviewed:1 }),
```
Comment (jimrandomh) [inline packages/lesswrong/lib/collections/posts/views.js]:
```suggestion
  augmentForDefaultView({ status:1, userId:1, reviewedByUserId:1, frontpageDate: 1, authorIsUnreviewed:1, createdAt: -1 }),
```

File: packages/lesswrong/lib/collections/comments/views.js (hunk 2 above)
Comment (Raemon) [reply packages/lesswrong/lib/collections/comments/views.js]: I didn't know how to account for the $or being blank. Does that mean the index just needs "userID: 1"?

**Pull request merged**